common/types/loadbalancer.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var log = logging.DefaultLogger
+var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "loadbalancer")
```

Comment:
Shouldn't this be `types`?

If there's loadbalancer code here we want to be part of a different subsystem, we should refactor it to `pkg/loadbalancer` or something.

Reply:
Good catch - IMO this should be moved anyway (I have this in a separate branch - will make) - the logging in this package is only for the loadbalancer. I can put out my PR to move it to a separate package soon :)